docs/docs/usage.md

```diff
@@ -7,58 +7,59 @@
 | help    | `mbt [command] --help` or<br> `mbt [command] -h`    | Prints detailed information on the selected command.                           
 <br>
 
-### How to know the tool's version
+### How to display the current tool version
```

Comment:
I do not quite like the "display" here.  Users do not want to display the version. They want to know what the version of the installed tool is.

Reply:
I understand. But "how to know a version" would be better rephrased. You are actually displaying it (printing) when you run the command. Maybe better: "How to find out the tool version"

Reply:
Like it. I will update.

Reply:
Ok, thanks.